gatsby-plugin-material-ui/src/gatsby-ssr.js

```diff
@@ -16,10 +17,13 @@ const defaultOptions = {
 };
 
 export const wrapRootElement = ({ element, pathname }, pluginOptions) => {
-  const sheets = new ServerStyleSheets(pluginOptions.stylesProvider);
-  globalLeak.set(pathname, sheets);
+  if (pluginOptions.stylesProvider || pluginOptions.pathToStylesProvider) {
```

Comment:
Do we need this branch? Shouldn't it throw if nothing is provided?

Reply:
@hupe1980 noted above that `gatsby-browser.js` initially had the ability of being used without plugin options. I already had that functionality implemented in `gatsby-node.js`, but on Hupe's note I added it back into `gatsby-browser.js`. To ensure ssr and browser works the same, I also added the conditional to the ssr as seen above.

Regardless on what we decide, both ssr and browser should probably work the same way?

Reply:
> ssr and browser should probably work the same way?

I agree!